source/game/system/map/MapdataCheckPoint.cc

```diff
@@ -137,9 +137,59 @@ void MapdataCheckPoint::clearSearched() {
     m_searched = false;
 }
 
+/// @addr{0x80511EC8}
+/// @brief Finds the offset between the two positions that enter the checkpoint.
+/// @details This assumes the player is entering the checkpoint as intended, and not from the side.
```

Comment:
What happens if the player enters from the side? Maybe we should briefly describe here what behavior would occur

Reply:
I think it's too difficult to explain briefly, especially without visuals.